dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -17,74 +17,132 @@
-    Note:
-    These inputs are fed to the function for estimating the outcome variable. The same set of required_variables is used for each
-    instance of an internal function.
-
-    - 'bucket_size_scale_factor': float, DummyOutcomeRefuter.DEFAULT_BUCKET_SCALE_FACTOR by default
-    For continuous data, the scale factor helps us scale the size of the bucket used on the data
-    Note: 
-    The number of buckets is given by: 
-        (max value - min value)
-        ------------------------
-        (scale_factor * std_dev)
-
-    - 'min_data_point_threshold': int, DummyOutcomeRefuter.MIN_DATA_POINT_THRESHOLD by default
-    The minimum number of data points for an estimator to run.
+    Attributes:
+        num_simulations (int, optional): The number of simulations to be run. 
+            This defaults to ``CausalRefuter.DEFAULT_NUM_SIMULATIONS``
+
+        transformation_list (list, optional): The transformation_list is a list of actions to be performed to obtain the outcome.
+            ``DummyOutcomeRefuter.DEFAULT_TRANSFORMATION`` is the default transformation, in which the dummy outcome is white noise.
+            
+            Each of the actions within a transformation is one of the following types:
+
+            * function argument: function ``pd.Dataframe -> np.ndarray``
+              
+              It takes in a function that takes the input data frame as the input and outputs the outcome
+              variable. This allows us to create an output varable that only depends on the covariates and does not depend 
+              on the treatment variable.
+
+            * string argument
+
+                * Currently it supports some common estimators like
+
+                    1. Linear Regression
+                    2. K Nearest Neighbours
+                    3. Support Vector Machine
+                    4. Neural Network
+                    5. Random Forest
+                
+                * Or functions such as:
+
+                    1. Permute
+                      This permutes the rows of the outcome, disassociating any effect of the treatment on the outcome.
+                    2. Noise
+                      This adds white noise to the outcome with white noise, reducing any causal relationship with the treatment.
+                    3. Zero
+                      It replaces all the values in the outcome by zero
+            
+            Examples:
+
+            The transformation_list is of the following form:
+
+            * If the function ``pd.Dataframe -> np.ndarray`` is already defined.
+              ``[(func,func_params),('permute', {'permute_fraction': val} ), ('noise', {'std_dev': val} )]``
+            * If a function from the above list is used
+              ``[('knn',{'n_neighbors':5}), ('permute', {'permute_fraction': val} ), ('noise', {'std_dev': val} )]``
+
+        true_causal_effect (function): A function that is used used to get the True Causal Effect for the modelled dummy outcome. 
```

Comment:
need to mention the function signature that you expect for this method. how many parameters? of what type?